content/docs/2.1/scalers/influxdb.md

```diff
@@ -0,0 +1,73 @@
++++
+title = "InfluxDB"
+layout = "scaler"
+availability = "v2.1+"
+maintainer = "Community"
+description = "Scale applications based on InfluxDB queries"
++++
+
+### Trigger Specification
+
+This specification describes the `InfluxDB` trigger that scales based on the results of a flux query.
+
+```yaml
+triggers:
+  - type: influxdb
+    metadata:
+      serverURL: http://influxdb:8086
+      organizationName: influx-org
+      organizationNameFromEnv: INFLUXDB_ORG_NAME # Optional: You can use this instead of `organizationName` parameter. See details in "Parameter List" section
+      thresholdValue: '4'
+      query: |
+        from(bucket: "bucket-of-interest")
+        |> range(start: -12h)
+        |> filter(fn: (r) => r._measurement == "stat")
+      authToken: some-auth-token
+      authTokenFromEnv: INFLUXDB_AUTH_TOKEN # Optional: You can use this instead of `authToken` parameter. See details in "Parameter List" section
+      metricName: influx-metric # Optional: This value will default to a unique value in source if not provided as a parameter
+```
+
+**Parameter list:**
+
+- `authToken` field holds the authentication token needed for the InfluxDB client to communicate with an associated server. 
+- `authTokenFromEnv` defines the authorization token, similar to `authToken`, but reads it from an environment variable on the scale target.
+- `organizationName` is the organization name needed for the client to locate all information contained in that [organization](https://docs.influxdata.com/influxdb/v2.0/organizations/) such as buckets, tasks, etc.
+- `organizationNameFromEnv` defines the organization name, similar to `organizationName`, but reads it from an environment variable on the scale target.
+- `serverURL` holds the url value of the InfluxDB server.
+- `thresholdValue` is provided by the user. This value can vary from use case to use case depending on the data of interest, and is needed to trigger the scaling in/out depending on what value comes back from the query.
+- `query` is the flux query that will yield the value for the scaler to compare the `thresholdValue` against.
+- `metricName` is an optional parameter that a user can give to provide some uniqueness of `ScaledObject` names. It's optional and will default in source code if not provided as a parameter.
```

Comment:
Minor nit here, could you please use the similar description (and examples) for `metricName` that's being used in this PR: https://github.com/kedacore/keda-docs/pull/313